packages/example-generator/src/run.ts

```diff
@@ -209,10 +215,22 @@ const UPDATE_README: Mutator = {
   }),
 };
 
+const UPDATE_TUTORIAL_IMPORTS: Mutator = {
```

Comment:
If we do always apply this:

```suggestion
// We re-export ontology object submodules from the root in this locally generated SDK
const UPDATE_SUBMODULE_IMPORTS: Mutator = {
```